non-oss/pharos_pro/addons/kontena-storage/addon.rb

```diff
@@ -101,10 +101,22 @@
   install {
     set_defaults
     cluster = build_cluster_resource
-    apply_resources(
-      cluster: cluster.to_h.deep_transform_keys(&:to_s),
-      rook_version: self.class.version.split('+').first
-    )
+    if upgrade_from?('0.8')
+      storage_stack = kube_stack(
+        cluster: cluster.to_h.deep_transform_keys(&:to_s),
+        rook_version: rook_version
+      )
+      logger.info "Applying upgrade ..."
+      storage_stack.apply(kube_client, prune: false)
```

Comment:
For some reason this will throw an error, which triggers retry, which triggers `else` 🤔 But everything still works after `pharos up` completes.